docs/developer-docs/latest/setup-deployment-guides/configurations/optional/typescript.md

```diff
@@ -0,0 +1,41 @@
+---
+title: TypeScript configuration - Strapi Developer Docs
+description: Details for the configuration of TypeScript projects
+sidebarDepth: 3
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/setup-deployment-guides/configurations/optional/sso.html
+---
+
+# TypeScript project configuration
+
+TypeScript-enabled Strapi applications have a specific [project structure](/developer-docs/latest/setup-deployment-guides/file-structure.md) with the following dedicated folders and configuration files:
```

Comment:
I would add a link to the development > typescript page on the "TypeScript" word.

Reply:
ok done